crates/rpc/rpc/src/eth/helpers/types.rs

```diff
@@ -1,13 +1,28 @@
 //! L1 `eth` API types.
 
-use alloy_network::Ethereum;
+use alloy_consensus::{ReceiptWithBloom, TxType};
+use reth_ethereum_primitives::Receipt;
 use reth_evm_ethereum::EthEvmConfig;
-use reth_rpc_convert::RpcConverter;
+use reth_rpc_convert::{RpcConverter, RpcTypes};
 use reth_rpc_eth_types::receipt::EthReceiptConverter;
 
+/// Ethereum specific RPC types.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, Hash)]
+#[non_exhaustive]
+pub struct EthereumRpcTypes;
+
+impl RpcTypes for EthereumRpcTypes {
+    type Header = alloy_rpc_types_eth::Header;
+    type Receipt = alloy_rpc_types_eth::TransactionReceipt<
+        ReceiptWithBloom<Receipt<TxType, alloy_rpc_types_eth::Log>>,
```

Comment:
could we add a sanity test case here just to make that this deserializes exactly the same?